ptf/saiswitch.py

```diff
@@ -17,12 +17,12 @@
 """
 from ipaddress import ip_address
 
-from sai_thrift.sai_headers import *
+from sai_thrift.sai_headers import *  # pylint: disable=wildcard-import; lgtm[py/polluting-import]
```

Comment:
If you use pylint, I guess there are some continued indentation issues and some lines are too long